### PR TITLE
Error in DataLoggerFiles and DataLoggerFileColumns #132

### PR DESCRIPTION
**Pull request by miguelcleon**
@lsetiawan I went ahead and made the fix for issue #132

 see that the first one, this is caused by Dataloggerfiles model, mapping to the wrong column within the database: https://github.com/ODM2/ODM2PythonAPI/blob/master/odm2api/ODM2/models.py#L458.

For the second one, measurementequation is indeed misspelled: https://github.com/ODM2/ODM2PythonAPI/blob/master/odm2api/ODM2/models.py#L514.

### CODE REVIEW

**Comment by miguelcleon** [comment]
why did this fail @lsetiawan @ocefpaf  ?

**Comment by miguelcleon** [comment]
also 

found two more problems with dataloggerfiles model 
There is no column `dataloggeroutputfiledescription` instead it is `dataloggerfiledescription`
https://github.com/ODM2/ODM2PythonAPI/blob/master/odm2api/ODM2/models.py#L460

There is no column `dataloggeroutputfilelink` instead it is `dataloggerfilelink`
https://github.com/ODM2/ODM2PythonAPI/blob/master/odm2api/ODM2/models.py#L461

**Comment by miguelcleon** [comment]
I can confirm instances of these models can now be retrieved without error.

**Comment by lsetiawan** [comment]
Sorry for replying to this so late. Was out sick yesterday. It seems like travis error is from Python 3.2 and 3.3 not being able to build the conda environment. Nothing to do with the changes. But, changes should go into the **development** branch rather than **master**. Please be advised. Thanks.

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,18 @@ language: python
 
 sudo: required
 
+# if the https://travis-ci.org/ODM2/ODM2PythonAPI/requests ever says: missing config
```
Comment (lsetiawan) [inline .travis.yml]: @miguelcleon Any reason why these were added? It seems like your changes may have broken Travis CI.

Comment (lsetiawan) [reply .travis.yml]: Any changes to Travis should go through a PR so it doesn't just break without anyone knowing why. Thanks. I am going to roll back your changes to fix Travis.

Comment (lsetiawan) [reply .travis.yml]: Your changes are now in PR, please remove any change in `.travis.yml`

**Comment by lsetiawan** [comment]
Moved to #134. Closing.

File: .travis.yml (hunk 1 above)
Comment (miguelcleon) [reply .travis.yml]: Yes this looks like a change I didn't mean to include.

**Pull request closed without merging**